admin/sync_logs.py

```
@@ -0,0 +1,217 @@
+    # Add query_hash column for better deduplication
+    try:
+        cursor.execute("ALTER TABLE query_logs ADD COLUMN query_hash TEXT UNIQUE")
+        print("Added query_hash column")
+    except sqlite3.OperationalError:
+        pass  # Column already exists
+
+    # Create index on query_hash
+    try:
+        cursor.execute("CREATE UNIQUE INDEX IF NOT EXISTS idx_query_logs_hash ON query_logs(query_hash)")
+    except sqlite3.OperationalError:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(sqlite3.OperationalError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(sqlite3.OperationalError)`

(SIM105)